app/src/pages/inside/testPlansPage/testPlansTable/testPlansTable.tsx

```diff
@@ -162,16 +183,52 @@ export const TestPlansTable = ({ testPlans }: TestPlansTableProps) => {
     },
   ];
 
+  const setTestPlansPage = (page: number): void => {
+    const offset = (page - 1) * testPlansPageData.size;
+
+    if (offset !== Number(query?.offset)) {
+      const url = `/organizations/${organizationSlug}/projects/${projectSlug}/testPlans?offset=${offset}&limit=${pageSize}`;
+      push(url);
+    }
+  };
+
+  const setTestPlansPageSize = (pageSize: number): void => {
+    if (pageSize !== Number(query?.limit)) {
+      const url = `/organizations/${organizationSlug}/projects/${projectSlug}/testPlans?offset=${defaultQueryParams.offset}&limit=${pageSize}`;
```

Comment:
I think we can create a function with two params `{ limit, offset }` and call it with diferent params - here and above

Reply:
Done